src/psyclone/tests/psyir/symbols/containersymbol_test.py

```diff
@@ -73,10 +78,12 @@ def test_containersymbol_initialisation():
     sym = ContainerSymbol("my_mod")
     assert isinstance(sym, ContainerSymbol)
     assert sym.name == "my_mod"
-    assert not sym._reference  # Reference are not evaluated until told
+    assert not sym._reference  # References are not evaluated until told
```

Comment:
As before please reword as it is not clear what the comment means.

Reply:
Done.